var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -291,6 +291,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=85835
     patch('sys_ustat.h.patch', when='@5.0:6.4,7.0:7.3,8.1')
     patch('sys_ustat-4.9.patch', when='@4.9')
+    patch('glibc-2.31-libsanitizer-3-gcc-5.patch', when='@5.3.0:5.5.0')
```

Comment:
There are a number of patches with a similar naming format around line 280: I think it would make sense to move this patch there.

Does this come from the same source (https://bugs.gentoo.org/708346) or did you create this patch yourself?

Reply:
>There are a number of patches with a similar naming format around line 280: I think it would make sense to move this patch there.

The order of patches is important which is why it was placed where it was. To move it up in the package recipe would require adjusting the other patches that affect gcc-5.

> Does this come from the same source (https://bugs.gentoo.org/708346) or did you create this patch yourself?

There are lots of "cyclades removal" patches floating around for gcc and llvm, generally for newer compiler versions. I created this one for gcc-5 but it is not specific to the referenced gentoo bug. I will add a comment and separate it from that block to be a little clearer.

Reply:
> The order of patches is important which is why it was placed where it was. To move it up in the package recipe would require adjusting the other patches that affect gcc-5.

That makes sense: it didn't initially look that way based on the patch names, but  `find var/spack/repos/builtin/packages/gcc/ -name "*.patch" | xargs grep -l -i "sanitizer"` does in fact show a few files that modify the same (or related) files.